### DISCUSSION TITLE
怎么找到方舟官方图床的？

### DISCUSSION THREAD

**Discussion by SherkeyXD** [Yue-plus/hexo-theme-arknights#49] [General]
比如博客首页的罗德岛标志
以及请求列出目前已知的图床（有空的话
十分感谢！

**Comment by Yue-plus** [comment]
浏览器右键审查元素。
目前没有全部已知图床。
----------------------------
另外，这毕竟也算是盗用了yj的cdn流量，如果有要求还是要改回本地的。
